xml-conduit/Text/XML/Stream/Parse.hs

```diff
@@ -406,23 +409,39 @@ data ParseSettings = ParseSettings
     -- Default: False
     --
     -- Since 1.2.1
+    , psPreserveWhiteSpace :: Bool
+    -- ^ Whether we should preserve literal XML whitespace within nodes.
+    --
+    -- Default: False
```

Comment:
This is a breaking change in behavior without a good reason. Why do we want to no longer preserve whitespace, and instead change the semantics of a document by default?